app/helpers/mergeable_record.rb

```diff
@@ -0,0 +1,43 @@
+class MergeableRecord
```

Comment:
This feels like it should be implemented as a "mergeable" model concern instead of a class you pass a model into.

Reply:
Also this seems unrelated to this PR. Let's try and avoid PRs that accomplish too many things.

Reply:
> This feels like it should be implemented as a "mergeable" model concern

Yup, that makes sense. Will do.

> Also this seems unrelated to this PR.

This came up when we were changing the merging functionality for the GET sync call, where we set `updated_on_server_at` for patients depending on the merged `status` of it's children. We refactored the `MergeRecord` module into this.